tests/inference_with_NaN_simulator_test.py

```diff
@@ -121,7 +121,7 @@ def linear_gaussian_nan(
     num_rounds = 2
 
     for r in range(num_rounds):
-        theta, x = simulate_for_sbi(simulator, proposals[-1], 550)
+        theta, x = simulate_for_sbi(simulator, proposals[-1], num_simulations)
```

Comment:
oh I see merge conflicts incoming :D

Reply:
yes, I will undo those and let you do it.